bundles/org.openhab.binding.smhi/src/main/java/org/openhab/binding/smhi/internal/SmhiBindingConstants.java

```diff
@@ -67,4 +68,6 @@
     public static final String APPROVED_TIME_URL = BASE_URL + "/api/category/pmp3g/version/2/approvedtime.json";
     public static final String POINT_FORECAST_URL = BASE_URL
             + "/api/category/pmp3g/version/2/geotype/point/lon/%.6f/lat/%.6f/data.json";
+
+    public static final BigDecimal OCTAS_TO_PERCENT = BigDecimal.valueOf(0.08);
```

Comment:
I think you are incorrectly trying to map the percent unit to be in the 0.0-1.0 range instead of the 0-100 range.
```suggestion
    public static final BigDecimal OCTAS_TO_PERCENT = BigDecimal.valueOf(8);
```

Reply:
No, e.g. 5/8 = 0.625, 5/0.08 = 62.5. 62.5 Is the number I want, right?

Reply:
Now that I think about it though, it might be more clear to just multiply by 12.5 instead.